lib/ansible/executor/module_common.py

```diff
@@ -940,6 +944,16 @@ def _find_module_utils(module_name, b_module_data, module_path, module_args, tas
         )))
         b_module_data = output.getvalue()
 
+ elif module_substyle == 'dcl':
+        # Start of DCL support
+        # shebang needs to start with $!....
+        shebang = u'$!'
```

Comment:
or we can move 'shebang' detection to the shell module (might also help with powershell @nitzmahone @jborean93 ?)